tests/unit/t_framefmt.c

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ * this file except in compliance with the License. A copy of the License is
+ * located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <aws/cryptosdk/default_cmm.h>
+#include <aws/cryptosdk/private/cipher.h>
+#include <aws/cryptosdk/private/framefmt.h>
+#include <aws/cryptosdk/session.h>
+#include <stdlib.h>
+#include "counting_keyring.h"
+#include "testing.h"
+#include "testutil.h"
+#include "zero_keyring.h"
+
+int test_serialize_return_ciphertext_size() {
+    // Arguments to the function
+    size_t ciphertext_buf_capacity = 10000;
+    size_t plaintext_size          = 100;
+    enum aws_cryptosdk_alg_id id   = ALG_AES128_GCM_IV12_TAG16_HKDF_SHA256_ECDSA_P256;
+
+    struct aws_allocator *alloc = aws_default_allocator();
+
+    // Alocate the frame
+    struct aws_cryptosdk_frame out_frame;
+    out_frame.type            = FRAME_TYPE_SINGLE;
+    out_frame.sequence_number = 1;
+
+    // Allocate the byte buffer
+    struct aws_byte_buf ciphertext_buf;
+    TEST_ASSERT_SUCCESS(aws_byte_buf_init(&ciphertext_buf, alloc, ciphertext_buf_capacity));
+
+    // Increase the length of the ciphertext to simulate having old data
```

Comment:
Okay, I'm just nitpicking about terminology now, but tests are supposed to communicate what the code is supposed to do. If we have confusing comments it may mislead future developers. When you say "old data" it makes it sound like there is stale data in the buffer that's no longer relevant. That's not what's going on here. What would be in the buffer already is the earlier part of the ciphertext message, for example the header or the previous serialized frame.

Change ``old data`` in this comment to ``data already in the buffer``.